src/main/php/PHPMD/Renderer/GitLabRenderer.php

```diff
@@ -61,7 +61,7 @@ protected function addViolationsToReport(Report $report)
                         'PHP',
                     ],
                 'check_name' => $violation->getRule()->getName(),
-                'fingerprint' => $violation->getFileName().':'.$violation->getBeginLine().':'.$violation->getRule(
+                'fingerprint' => $violation->getFileName() . ':' . $violation->getBeginLine() . ':' . $violation->getRule(
```

Comment:
To make phpcs also happy:
```suggestion
                'fingerprint' => $violation->getFileName()
                    . ':' . $violation->getBeginLine()
                    . ':' . $violation->getRule()->getName(),
```

Reply:
Already fixed in another way.
Thanks